src/libraries/Microsoft.PowerFx.Interpreter/Functions/Mutation/PatchFunction.cs

```diff
@@ -205,11 +209,11 @@ public override bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType
 
                 foreach (var typedName in curType.GetNames(DPath.Root))
                 {
-                    DName name = typedName.Name;
-
-                    if (!dataSourceType.TryGetType(name, out DType dsNameType))
+                    if (!(dataSourceType.DisplayNameProvider != null && (dataSourceType.DisplayNameProvider.TryGetDisplayName(typedName.Name, out var displayName) ||
```

Comment:
>dataSourceType.DisplayNameProvider [](http://example.com/codeflow?start=26&length=34)

Can we use this instead? 
[https://github.com/microsoft/Power-Fx/blob/dd90f041aa97f4b98ce440340017d455cd7cc8f3/src/libraries/Microsoft.PowerFx.Core/Public/Types/AggregateType.cs#L72](https://github.com/microsoft/Power-Fx/blob/dd90f041aa97f4b98ce440340017d455cd7cc8f3/src/libraries/Microsoft.PowerFx.Core/Public/Types/AggregateType.cs#L72)


Turns out that DisplayNameProvider is just one way we may check display names.